Specs/Sequence/English/EmailModel.json

```diff
@@ -142,13 +154,13 @@
   },
   {
     "Input": "Both a..bc@outlook.com and .abc@hotmail.com are not valid e-mail addresses.",
-	"Comment": "By default the current system is strict. If a relaxed match is needed (to catch these), enable the Relaxed option.",
+    "Comment": "By default the current system is strict. If a relaxed match is needed (to catch these), enable the Relaxed option.",
```

Comment:
what is this "Relaxed" option? I am thinking if we should enable this with the relaxed option itself?

Reply:
My proposal was not to enable these settings. Instead to enable the detection of email address not be recognized when there are some Chinese characters in the sentence as a relaxed option instead of by default

Reply:
> My proposal was not to enable these settings. Instead to enable the detection of email address not be recognized when there are some Chinese characters in the sentence as a relaxed option instead of by default

Relaxed option will not solve the issue, the issue is caused by the regex that I am updated in this PR.
There are 2 regex used to extract email, in case there is a Chinese sentence **"1邮件地址test@test.com"**, the first regex matches **"test@test.com"**, the 2nd regex (the regex I have fixed in the PR) matches **"1邮件地址test@test.com"**, so the results merged as **"1邮件地址test@test.com"**, if relaxed option is enabled here, it will directly return **"1邮件地址test@test.com"** which is incorrect, the correct email should be "test@test.com"

Reply:
Need to mention that currently we don't support RFC 6530 which allow non-ASCII characters in email address, such as Chinese, Arabic.

Reply:
I am not very familiar with this codebase but my idea was 

Using the relaxed option - run the regex without the word separator to return the correct detection for 邮件地址test@test.com along with all other existing anomalous behaviors like ending with a . or having multiple dots (..) in the email address and keep existing email detection intact so we do not break existing customers. 

That said, if this is indeed a bug fix, it should be ok to fix in the regular regex.

Reply:
The more that I think, I am ok with your approach.